saleor/core/permissions.py

```diff
@@ -157,3 +157,26 @@ def get_permissions_from_codenames(permission_codenames: List[str]):
         .prefetch_related("content_type")
         .order_by("codename")
     )
+
+
+def _permission_required(perms, context):
```

Comment:
I think the `context` can be an incorrect name. Based on your code I would say that this is a requestor: Union[User, App]

Reply:
You're using this function to check single permission one by one. Maybe we could change the name `perms` to `perm` and use `requestor.has_perm` method (this will simplify and increase readability for calling this function)
`if _permission_required((perm,), requestor):` -> `if _permission_required(perm, requestor):`